src/Http/BadgeApiClient.php

```diff
@@ -79,15 +78,15 @@ public function sendReport(
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
         curl_setopt($ch, CURLOPT_POST, true);
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-        curl_setopt($ch, CURLOPT_HTTPHEADER, iterator_to_array($this->createHeaders($apiKey)));
+        curl_setopt($ch, CURLOPT_HTTPHEADER, $this->createHeaders($apiKey));
```

Comment:
We're not on 7.4 yet, else it'll have been just `[...arrGen()]`

Reply:
That was precisely the way I developed it and then realised that I was tired and hadn't checked the project's PHP requirements, so that was my fix to get it working ;-)